pkg/util/exec/fake_exec.go

```diff
@@ -143,3 +145,69 @@ func (fake *FakeExitError) Exited() bool {
 func (fake *FakeExitError) ExitStatus() int {
 	return fake.Status
 }
+
+func commandAsStringSlice(command interface{}) []string {
+	if args, ok := command.([]string); ok {
+		return args
+	} else if commandstr, ok := command.(string); ok {
+		return strings.Split(commandstr, " ")
```

Comment:
I'm with @danwinship here, it will cause some arcane errors, but I believe the 90% use case will be easier.

Perhaps we can ban `"` characters in strings and return an error if someone tries to pass in `foo "bar blat"`?